lib/reporters/mini.js

```diff
@@ -70,19 +70,16 @@ MiniReporter.prototype.prefix = function (str) {
 };
 
 MiniReporter.prototype._test = function (test) {
-	var title;
+	var title = test.title;
 
 	if (test.todo) {
-		title = colors.todo('- ' + test.title);
```

Comment:
If a human can not see `todo` and `skip` tests, shouldn't we also remove color from there?